mule-extensions-api/src/main/java/org/mule/runtime/extension/internal/loader/util/InfrastructureParameterBuilder.java

```diff
@@ -248,7 +248,7 @@ public static ParameterDeclaration addErrorMappings(OperationDeclaration operati
         .allowsReferences(false)
         .allowTopLevelDefinition(false)
         .build());
-    parameter.addModelProperty(new QNameModelProperty(MULE_ERROR_MAPPING_QNAME));
+    parameter.addModelProperty(new QNameModelProperty(MULE_ERROR_MAPPINGS_QNAME));
```

Comment:
this is API. Can't rename it

Reply:
this was added in 4.4.x, it doesn;t exist in 4.3, so it is still `new`